views/index.ejs

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Index Page</title>
+</head>
+<body>
+	<ul class="missions">Mars Mission Index
+		<li><a href="http://localhost:3000/marsMissions/0">Curiosity</a></li>
```

Comment:
NO!! 
you should pass data to this template from the route when you render, and use a LOOP to 
* interpolate the links in for each page
* interpolate in the names of each mission (do not hard code them!)